source/install.txt

```diff
@@ -45,17 +45,18 @@ Steps
       Alternatively, you can press :kbd:`Cmd` + :kbd:`,` on macOS or 
       :kbd:`Ctrl` + :kbd:`Alt` + :kbd:`S` on Windows and Linux.
 
-   .. step:: Show the Plugins tab. 
+   .. step:: Show the Plugins settings. 
```

Comment:
for consistency, changed references to the plugins "tab" to "settings" so there's distinction from the "marketplace" & "installed" tabs